lesson_10/tests/test_event_loggin.py

```diff
@@ -0,0 +1,46 @@
+import os
+import logging
+import pytest
+from lesson_10.src.functions import log_event
+
+def test_log_event_success():
+    # clear the logger before starting the test
+    for handler in logging.root.handlers[:]:
+        logging.root.removeHandler(handler)
+
+    # call the function that records the log
+    log_event("test_user", "success")
+
+    # check if the file has been created
+    assert os.path.exists('login_system.log')
+
+    # check if the file contains the expected message
+    with open('login_system.log', 'r') as f:
+        logs = f.read()
+        assert "Login event - Username: test_user, Status: success" in logs
```

Comment:
there are many duplications in tests, try to create a separate function for repetitions